python/ray/data/tests/test_image.py

```diff
@@ -263,6 +264,23 @@ def test_unidentified_image_error(ray_start_regular_shared):
                 ray.data.read_images(paths=file.name).materialize()
 
 
+class TestWriteImages:
+    @pytest.mark.parametrize("file_format", [None, "png"])
+    def test_write_images(ray_start_regular_shared, file_format, tmp_path):
+        ds = ray.data.read_images("example://image-datasets/simple")
+        ds.write_images(
```

Comment:
Can we also test followed cases:
* the dataset has multiple columns
* the provided file format is illegal
* `write_images()`, then `read_images()` back.